pkg/operator/ceph/object/notification/provisioner.go

```diff
@@ -166,21 +167,50 @@ var createNotification = func(p provisioner, bucket *bktv1alpha1.ObjectBucket, t
 }
 
 // Allow overriding this function for unit tests
-var deleteAllNotificationsFunc = deleteAllNotifications
+var getAllNotificationsFunc = getAllNotifications
 
-var deleteAllNotifications = func(p provisioner, bucket *bktv1alpha1.ObjectBucket) error {
+var getAllNotifications = func(p provisioner, ob *bktv1alpha1.ObjectBucket) ([]string, error) {
```

Comment:
I had a bit of trouble following what some values were, and I think this function's name could be improved to help readability. This lists the notifications on a bucket as seen by the RGW, correct?

If so, maybe this would be better interpreted as `getAllRGWNotifications` (or similare).
If not, maybe this would be better interpreted as `getAllNotificationResources` (or similar).